### ISSUE TITLE
Downloading large files times out

### ISSUE THREAD

**Issue by jonahkagan** [votingworks/arlo#1616]
Currently the server proxies the download to S3, but if the request takes too long (>30s) it gets timed out by Heroku. We may need to rethink this (e.g. maybe use direct S3 download links)

**Comment by mattroe** [comment]
@jonahkagan do we have data on how often AAs download the CVR once uploaded?

**Comment by jonahkagan** [comment]
No, we don't really have data on this. I don't think it's super common unless something goes wrong in the audit. @ginvdr might have a better intuition.

**Comment by ginvdr** [comment]
I don't think it's use often now.  Nevada and Washington would be the users who would.  CVRs aren't likely to have errors so I think the more likely reason an audit admin would download the file is to share it publicly.  Washington cannot share CVRs publicly and Nevada hasn't progressed far enough in the process to do so.
